server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -78,11 +78,16 @@ export class TenantManager {
             return Promise.reject(new Error("Tenant is disabled or does not exist."));
         }
 
+        const customData = { ...tenant.customData };
```

Comment:
Why do we need a "spread" operator here?

Reply:
I misunderstood about the some best practice of JavaScript
Will remove this code in the next revision.